contrib/fsl_integration/sct_plugin.py

```diff
@@ -0,0 +1,267 @@
+# coding=utf-8
+
+#########################################################################################
+# This code will provide SCT integration into fsleyes for the following tools:
+#
+#    - sct_deepseg_gm
+#    - sct_deepseg_sc
+#
+# ---------------------------------------------------------------------------------------
+# Copyright (c) 2018 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Christian S. Perone
+# Created: 2 Mar 2018
+#########################################################################################
+
+import os
+import subprocess
+from threading import Thread
+
+import wx
+import wx.lib.agw.aui as aui
+import wx.html as html
+
+
+aui_manager = frame.getAuiManager()
+
+
+def sct_call(command):
```

Comment:
This function is a copy of line 43. Is this function needed?

Reply:
Good catch, I removed it. Thanks !